commands/publish/lib/get-unpublished-packages.js

```diff
@@ -1,33 +1,30 @@
 "use strict";
 
 const log = require("libnpm/log");
-const pReduce = require("p-reduce");
+const pMap = require("p-map");
```

Comment:
I removed `p-reduce` from the main command script, forgot it was here, and then realized it could be implemented with more concurrency with `p-map`. (`p-reduce` executes serially)